editor/src/interaction/terrain.rs

```diff
@@ -116,11 +117,17 @@ impl BrushGizmo {
     }
 }
 
-fn copy_layer_masks(terrain: &Terrain, layer: usize) -> Vec<Vec<u8>> {
+fn copy_layer_masks(terrain: &Terrain, layer: usize, ui: &mut UserInterface) -> Vec<Vec<u8>> {
```

Comment:
This should be in a separate pull request, because it does not have anything in common with ABSM editor.